### PR TITLE
Embedding video, adding PS tag definitions

### PR DESCRIPTION
**Pull request by idanenglander**
@fpliger If you could please assist in the defining those visual tags and py-repl that would be much appreciated.

Also, the video I'm embedding is here: https://www.youtube.com/watch?v=ZtC7TCt_LhU 
As you can see it is public. If you want that changed let me know.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "sphinx_togglebutton",
     "sphinx_sitemap",
     "sphinxemoji.sphinxemoji",
+    "sphinxcontrib.youtube",
```
Comment (kathatherine) [inline docs/conf.py:38]: You'll also need to add this to the environment.yml, I believe.

File: docs/tutorials/getting-started.md [hunk 2]
```diff
@@ -207,3 +207,18 @@ fig
   </body>
 </html>
 ```
+## The py-repl tag
+
+The `<py-repl>` tag creates a REPL component that is rendered to the page as a code editor, allowing you to write executable code inline.
+
+## Visual components
+
+The following tags control visual attributes of your html page.
```
Comment (kathatherine) [inline docs/tutorials/getting-started.md:216]: Capital HTML, for consistency.

**Review by kathatherine — CHANGES_REQUESTED** [review]

**Review by kathatherine — APPROVED** [review]
LGTM!

File: docs/environment.yml [hunk 3]
```diff
@@ -10,7 +10,9 @@ dependencies:
 -   sphinx-copybutton
 -   sphinx-design
 -   sphinx-togglebutton
+
 -   pip:
     -   sphinxemoji
     -   sphinx-sitemap
     -   sphinx-autobuild
+    -   sphinxcontrib.youtube
```
Comment (jezdez) [inline docs/environment.yml:18]: The Python module that you import as part of the Sphinx configuration is using the Python dotted import notation `sphinxcontrib.youtube`, but that project to install that contains it is called [sphinxcontrib-youtube](https://pypi.org/project/sphinxcontrib-youtube/).

```suggestion
    -   sphinxcontrib-youtube
```

**Review by jezdez — APPROVED** [review]
Just a correction on the package name, but that's all.

File: docs/environment.yml (hunk 3 above)
Comment (jezdez) [reply docs/environment.yml:18]: @idanenglander You can use the "Commit suggestion" feature above to apply the suggestion if you'd like, or do the change manually, whatever works for you best.

Comment (idanenglander) [reply docs/environment.yml:18]: So it seems to be complaining about that when I try building with sphinxcontrib-youtube... but not when I use the period.

`Could not import extension sphinxcontrib-youtube (exception: No module named 'sphinxcontrib-youtube')
make: *** [html] Error 2`

Comment (jezdez) [reply docs/environment.yml:18]: It should be `sphinxcontrib-youtube` in environment.yml and `sphinxcontrib.youtube` in conf.py, then this error should go away.

Comment (idanenglander) [reply docs/environment.yml:18]: huzzah! You are correct. Thank you

**Review by fpliger — APPROVED** [review]
Thanks @idanenglander !! It lgtm! How about we merge as it is and I help with the visual components in follow up PRs?

**Pull request merged**